examples/tree_1d_dgsem/elixir_shallowwater_multilayer_dam_break_es_dry.jl

```diff
@@ -119,6 +119,7 @@ stage_limiter! = PositivityPreservingLimiterShallowWater(variables = (Trixi.wate
 ###############################################################################
 # run the simulation
 
-sol = solve(ode, SSPRK43(stage_limiter!), dt = 1.0,
-            save_everystep = false, callback = callbacks, adaptive = false);
+sol = solve(ode, SSPRK43(stage_limiter!)
+            dt = 1.0, adaptive = false,
+            ode_default_options()...,, callback = callbacks);
```

Comment:
```suggestion
sol = solve(ode, SSPRK43(stage_limiter!),
            dt = 1.0, adaptive = false,
            ode_default_options()..., callback = callbacks);
```
Thanks for updating this! I noticed that the test for this elixir is failing due to a misplaced comma.

Reply:
```suggestion
sol = solve(ode, SSPRK43(stage_limiter!);
            dt = 1.0, adaptive = false,
            ode_default_options()..., callback = callbacks);
```